test/Sniffs/Commenting/FileLevelDocBlockSniffTest.php

```diff
@@ -0,0 +1,110 @@
+<?php
+/**
+ * @see       https://github.com/zendframework/zend-coding-standard for the canonical source repository
+ * @copyright https://github.com/zendframework/zend-coding-standard/blob/master/COPYING.md Copyright
+ * @license   https://github.com/zendframework/zend-coding-standard/blob/master/LICENSE.md New BSD License
+ */
+
+namespace ZendTest\CodingStandard\Sniffs\Commenting;
+
+use ZendTest\CodingStandard\SniffTestCase;
+
+class FileLevelDocBlockSniffTest extends SniffTestCase
+{
+    /**
+     * @dataProvider assetsProvider
+     */
+    public function testAssets($asset, $fixed, $errorCount, array $errors, $warningCount, array $warnings)
+    {
+        $result = $this->processAsset($asset);
+
+        $this->assertErrorCount($errorCount, $result);
+        $this->assertWarningCount($warningCount, $result);
+
+        foreach ($errors as $error) {
+            $this->assertHasError($error, $result);
+        }
+
+        foreach ($warnings as $warning) {
+            $this->assertHasWarning($warning, $result);
+        }
+
+        $this->assertAssetCanBeFixed($fixed, $result);
+    }
+
+    public function assetsProvider()
+    {
+        return [
+            'FileLevelDocBlockPass'                   => [
```

Comment:
Don't pad the `=>` declaration here or with siblings to this item. Since it's a multi-line array declaration, alignment with other items of the same nesting level isn't necessary, and actually distracts.